api/common/v1/common.proto

```diff
@@ -68,6 +68,8 @@ message Run {
   google.protobuf.Timestamp scheduled_at = 10; // required
   google.protobuf.Timestamp started_at = 11;
   google.protobuf.Timestamp finished_at = 12;
+
+  map <string,string> computed_data = 13;
```

Comment:
Okay if we move this up to field 7? I'm intentionally keeping all the timestamp related stuff in fields 10-19 (arbitrary decision, but figured now that it's there let's stick with it).